src/Main.java

```diff
@@ -12,55 +12,65 @@ public static void task1(){
         if (year < 1584){
             System.out.println("Данные заведены не корректно");
         }
-        String leapYear = highYearCalculation(year);
-        System.out.println(year + leapYear);
+        boolean leapYear = definitionOfHighYear(year);
+        if (leapYear){
+            System.out.println(year + " год — високосный год");
+        }else{
+            System.out.println(year + " год — невисокосный год");
+        }
     }
-    public static String highYearCalculation(int year){
-        if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0 && year > 1584)){
-            return " год — високосный год";
-        }else
-            return " год — невисокосный год";
+    public static boolean definitionOfHighYear(int year){
+        boolean flag;
+        if (year > 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0){
+            return flag = true;
+        }else {
+            return flag = false;
+
+        }
```

Comment:
упрощается до 1 строки
```suggestion
        return year > 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```